data_collection/timeprof_matrix_bot.py

```diff
@@ -478,24 +460,45 @@ async def run_at(self, dt, coro):
         await self.wait_until(dt)
         return await coro
 
-    def create_next_sample_time(self, prev_sample_time, rate):
+    def draw_next_ping_time(self, prev_sample_time, rate):
         interval = np.random.exponential(scale=rate)
         next_sample_time = prev_sample_time + timedelta(minutes=interval)
         return next_sample_time
 
-    def schedule_next_sample(self, user_id, sample_time):
+    async def ping(self, user_id, sample_time):
```

Comment:
ping_time instead of sample_time to be consistent?

Reply:
yes